Source/WebKit/NetworkProcess/NetworkProcess.h

```diff
@@ -266,6 +266,9 @@ class NetworkProcess : public AuxiliaryProcess, private DownloadManager::Client,
 #if ENABLE(APP_BOUND_DOMAINS)
     void setAppBoundDomainsForResourceLoadStatistics(PAL::SessionID, HashSet<WebCore::RegistrableDomain>&&, CompletionHandler<void()>&&);
 #endif
+#if ENABLE(MANAGED_DOMAINS)
+    void setManagedDomainsForResourceLoadStatistics(PAL::SessionID, HashSet<WebCore::RegistrableDomain>&&, CompletionHandler<void()>&&);
+#endif // ENABLE(MANAGED_DOMAINS)
```

Comment:
These end comments are not great when the `#if` statement just covers a few lines. I see those throughout the patch, but not in the pre-existing code nearby.